### ISSUE TITLE
translation not working

### ISSUE THREAD

**Issue by OskarStark** [sonata-project/SonataUserBundle#621]
Here we use this translation in ```de```:

```xml
            <trans-unit id="Bad credentials">
                <source>Bad credentials.</source>
                <target>Benutzername oder Passwort ungültig</target>
            </trans-unit>
```
Ref: https://github.com/sonata-project/SonataUserBundle/blob/master/Resources/translations/SonataUserBundle.de.xliff#L497

This isn't working, because of the used translation_domain (FOSUserBundle) here:
```
            {% block sonata_user_login_error %}
                {% if error %}
                    <div class="alert alert-danger">{{ error|trans({}, 'FOSUserBundle') }}</div>
                {% endif %}
            {% endblock %}
```

but how to fix this problem, i created a ```FOSUserBundle.de.xliff``` file in ```app/Resources/translations```, but how to fix this for this bundle?

Oskar

**Comment by ejkun** [comment]
I was looking in FOSUserBundle latest commits about this. They took out the `Bad Credentials.` message, changed the `SecurityController` and changed the `login.html.twig` file, all in this [commit](https://github.com/FriendsOfSymfony/FOSUserBundle/commit/848884313726c81a42f86c0952957872d3c532e2). 

All of this should be fixed with the FOSUserBundle 2.0 version. But IMO, it will take a while. 
I managed to fix the error doing the same changes they made. 
But with the controller, we will have to copy they function and change the lines about the `$error` variable. I don't have time to do it right now, but later today or tomorrow I will make the changes in my fork and see if it pass all the tests.

**Comment by ejkun** [comment]
As I said in the last comment, I found a way to fix it. But since there were many changes in the repository structure, the PR was closed, but the problem still there. @OskarStark , should I rebuild the PR for the 3.x branch? Beacuse it's just a quickfix, if a stable 2.x version of FOSUserBundle is released, everything done in the PR will have to be removed, since the controller itself would stop calling FOSUserBundle controller. What do you think?
